fhir-examples/pom.xml

```diff
@@ -32,4 +32,195 @@
             </plugin>
         </plugins>
     </build>
+
+    
+    <profiles> 
+        <profile>
+            <id>fhir-javadocs</id>
+            <build>
+                <plugins>
+                    <!-- limit the javadoc plugin to jenkins only -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <version>3.1.1</version>
+                        <configuration>
+                            <verbose>false</verbose>
+                            <doclint>none</doclint>
+                            <additionalparam>-Xdoclint:none</additionalparam>
+                            <minmemory>128m</minmemory>
+                            <maxmemory>1g</maxmemory>
+                            <archive>
+                                <manifestEntries>
+                                    <Automatic-Module-Name>com.ibm.fhir</Automatic-Module-Name>
+                                </manifestEntries>
+                            </archive>
+                            <destDir>target/apidocs/${project.artifactId}</destDir>
+                            <additionalOptions>
+                                <additionalOption>--frames</additionalOption>
+                            </additionalOptions>
+                            <bottom><![CDATA[Copyright 2019, <a href="http://www.ibm.com">IBM Corporation<a><br></br>FHIR&reg; is the registered trademark of HL7 and is used with the permission of HL7.]]></bottom>
```

Comment:
super minor, but `<br></br>` could just be `<br />`

Reply:
yeah - I did not find that worked all the time in the build, so I worked around it.  by being explicit.

Reply:
I don't think `<br></br>` is more explicit.  using just `<br>` is valid html.  And since I know you like the balance of XHTML, i suggested `<br />`.  That this would break Gatsby is just bizaar